ui/app/styles/core/buttons.scss

```diff
@@ -14,10 +14,9 @@
   display: inline-block;
   font-size: $size-8;
   font-weight: $font-weight-semibold;
-  height: 2.5rem;
-  line-height: 1.6;
+  min-height: 2.25rem;
   min-width: 6rem;
-  padding: $size-10 $size-8;
+  padding: 0.5625rem 0.9375rem;
```

Comment:
This universally matches our button dimensions to HDS (instead of matching HDS to our buttons and then later removing the modification) until we have the chance to replace all of our structure buttons (coming soooon!! 🔜 )